src/features/applicationList/ApplicationList.tsx

```diff
@@ -164,6 +176,32 @@ const ApplicationList = ({
             />
           );
         }}
+        renderTableToolsTop={({ selectedRows }, { resetSelectedRows }) => {
+          const offers = getDraftedOffers(selectedRows);
```

Comment:
Same comment here as in unmarked ws notice list: the additional logic could be moved to the `ApplicationListTools` component?